classes/log.php

```diff
@@ -19,10 +19,10 @@ public static function info($msg, $method = null)
 		return static::write(\Fuel::L_INFO, $msg, $method);
 	}
 
-	public static function write($level, $msg, $method = null){
+	public static function write($level, $msg, $method = null) {
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Opening brace should be on a new line